diagnostic.md

```diff
@@ -9,6 +9,15 @@ Assume all code executes in strict mode.
 
 Please list the JavaScript primitives and give one example of each.
 
+ Number, String, Boolean, Undefined, Null.
+
+ Examples:
+ Number: 3
+ String: 'hello'
+ Boolean: true
+ Undefined: television
```

Comment:
Totally makes sense. I was basically thinking about it in the sense that I had not yet defined "television", but I did talk to Rachel about this in my 1 on 1 today.